crates/rattler_networking/src/authentication_storage/storage.rs

```diff
@@ -190,4 +135,20 @@ impl AuthenticationStorage {
             Ok((url, None))
         }
     }
+
+    /// Delete the authentication information for the given host
+    pub fn delete(&self, host: &str) -> Result<()> {
+        {
+            let mut cache = self.cache.lock().unwrap();
+            cache.insert(host.to_string(), None);
+        }
+
+        for backend in &self.backends {
+            if let Err(e) = backend.delete(host) {
+                tracing::warn!("Error deleting credentials from backend: {}", e);
+            }
+        }
+
+        Ok(())
```

Comment:
Shouldnt this return an error if it failed?

Reply:
What does "fail" mean here? An attempt is made to delete the credentials in _all_ known backends, regardless of whether the previous backend returned an error.

Reply:
Well each individual backend might fail but this function will still report success. I think if any of them fail the function should return an error too.

Reply:
But then the fallback behavior differs from `store/get`, where errors are only logged and the next backend is used. What do you think about returning an error if all backends fail, which would be consistent with the other methods?

Reply:
Yeah that sounds good! Maybe that should also be the behavior for the other methods?